### PR TITLE
#1 Add Admin ability to add time_ranges

### PR DESCRIPTION
**Pull request by alexrudall**
- Add admin controller, views, translations for time_range CRUD
- Sort translations file alphabetically

Screenshot 2020-10-20 at 15 24 27

### CODE REVIEW

File: spec/features/admin/time_ranges/destroy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  before { log_in create(:admin) }
+
+  it 'destroys time range' do
+    visit admin_time_ranges_path
+
+    accept_confirm { first('.bi-trash').click }
+
+    expect(page).to have_content(I18n.t('time_range.notice.successfully.destroyed'))
+    refute(TimeRange.exists?)
```
Comment (barbarawuk) [inline spec/features/admin/time_ranges/destroy_spec.rb:14]: haven't used refute before, nice :thumbsup:

File: app/controllers/admin/time_ranges_controller.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      render action: :edit
+    end
+
+    def create
+      @time_range = TimeRange.new(time_range_params)
+      if @time_range.save
+        redirect_to admin_time_ranges_path, notice: t('time_range.notice.successfully.created')
+      else
+        flash.now.alert = t('time_range.notice.could_not_be.created')
+        render :edit
```
Comment (barbarawuk) [inline app/controllers/admin/time_ranges_controller.rb:18]: Slight dip in coverage which could be avoided with a test for creating/editing invalid data e.g. with end after start :grinning:

**Review by barbarawuk — CHANGES_REQUESTED** [review]
Quite difficult to use this page to see the data for a single user or data type (a search/filter would be nice), but this will do the job for manual input of data until we have data integration via CSV or API done 👍

File: app/views/admin/time_ranges/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                <%= I18n.t('time_range.labels.user') %>
+              </th>
+              <th style='width: 50px;'></th>
+              <th style='width: 50px;'></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @time_ranges.each do |time_range| %>
+              <% cache time_range do %>
+                <tr>
```
[the remark this answers is not in the document]
Comment (alexrudall) [reply app/views/admin/time_ranges/index.html.erb:36]: Good spot. Added cache busting to User model to sort this.

File: app/controllers/admin/time_ranges_controller.rb (hunk 2 above)
Comment (alexrudall) [reply app/controllers/admin/time_ranges_controller.rb:18]: Thanks, forgot to check this.

**Pull request closed without merging**